### ISSUE TITLE
Installation script broken

### ISSUE THREAD

**Issue by daef** [ej52/proxmox-scripts#126]
**Script**
[npm_install.sh](https://github.com/ej52/proxmox-scripts/blob/main/lxc/nginx-proxy-manager/install/alpine.sh)

**Describe the bug**
Installations fails with `'rU' while trying to load binding.gyp`

**System info:**
 - Proxmox Version: 8.0.3
 - Alpine Version: 3.18

**Additional context**
Seems like a python 3.11 compat problem with gyp 7.
Also see https://github.com/ej52/proxmox-scripts/issues/125#issuecomment-1634971278

**Comment by JVKeller** [comment]
Any updates on this? Alpine 3.16 template is not showing in PVE 8.0.3, so 3.18 seems to be my only option at the moment.

**Comment by daef** [comment]
no, I gave up - I've got told that NPM is supposed to make my life easier - all I got was a headache

**Comment by Eeems** [comment]
> no, I gave up - I've got told that NPM is supposed to make my life easier - all I got was a headache

This is not npm, this is just a script someone made to automate installing it into a proxmox container.

**Comment by craeckor** [comment]
My friend still had Alpine 3.16 as an option for templates.
I uploaded the template to my CDN just in case it gets deleted: [https://cdn1.craeckor.ch/data/alpine-3.16-default_20220622_amd64.tar.xz](https://cdn1.craeckor.ch/data/alpine-3.16-default_20220622_amd64.tar.xz)
It works with Nginx Proxy Manager v2.9.14 (tested), i didn't tested newer versions.
